src/main/resources/assets/clientcommands/lang/en_us.json

```diff
@@ -266,36 +266,35 @@
   "commands.cvillager.noProfession": "The targeted villager has no profession",
   "commands.cvillager.notAVillager": "Target was not a villager",
   "commands.cvillager.notLevel1": "The targeted villager has level above 1",
-  "commands.cvillager.uncracked": "Your villager's RNG is not cracked (~18 quintillion seeds remain)",
   "commands.cvillager.partiallyCracked": "Your villager's RNG is partially cracked (~83m seeds remain)",
   "commands.cvillager.removeGoal.invalidIndex": "Unable to remove goal %s, you only have %s goals",
   "commands.cvillager.removeGoal.success": "Successfully removed goal %s",
   "commands.cvillager.resetCracker": "Successfully reset cracked villager RNG",
   "commands.cvillager.success": "Got the correct trade with correction of %sms",
   "commands.cvillager.target.cleared": "Target entity cleared",
   "commands.cvillager.target.set": "Target entity set",
+  "commands.cvillager.uncracked": "Your villager's RNG is not cracked (~18 quintillion seeds remain)",
 
+  "villagerManip.help.day": "Help: Villager RNG manipulation can only be done at night-time",
```

Comment:
The `villagerManip` translations shouldn't be in the middle of the commands, as that's not alphabetical

Reply:
ok, i'll move them